### ISSUE TITLE
NullReferenceException for Net Core 2.1, SDK 2.1.302

### ISSUE THREAD

**Issue by gitorrin** [jerriep/dotnet-outdated#56]
On Linux (Ubuntu 14.04.5), it seems the command fails. 
**dotnet --info**
.NET Core SDK (reflecting any global.json):
 Version:   2.1.302
 Commit:    9048955601

Runtime Environment:
 OS Name:     ubuntu
 OS Version:  14.04
 OS Platform: Linux
 RID:         ubuntu.14.04-x64
 Base Path:   /usr/share/dotnet/sdk/2.1.302/

Host (useful for support):
  Version: 2.1.2
  Commit:  811c3ce6c0

.NET Core SDKs installed:
  2.1.302 [/usr/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.2 [/usr/share/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.2 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.2 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

**Exception:**
_dotnet outdated_
Unhandled Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at DotNetOutdated.Services.ProjectAnalysisService.AnalyzeProject(String projectPath, Boolean includeTransitiveDependencies, Int32 transitiveDepth) in C:\projects\dotnet-outdated\src\DotNetOutdated\Services\ProjectAnalysisService.cs:line 24
   at DotNetOutdated.Program.OnExecute(CommandLineApplication app, IConsole console) in C:\projects\dotnet-outdated\src\DotNetOutdated\Program.cs:line 136
   at McMaster.Extensions.CommandLineUtils.Conventions.ExecuteMethodConvention.InvokeAsync(MethodInfo method, Object instance, Object[] arguments) in C:\projects\commandlineutils\src\CommandLineUtils\Conventions\ExecuteMethodConvention.cs:line 77
   at McMaster.Extensions.CommandLineUtils.Conventions.ExecuteMethodConvention.OnExecute(ConventionContext context) in C:\projects\commandlineutils\src\CommandLineUtils\Conventions\ExecuteMethodConvention.cs:line 62
   at McMaster.Extensions.CommandLineUtils.Conventions.ExecuteMethodConvention.<>c__DisplayClass0_0.<<Apply>b__0>d.MoveNext() in C:\projects\commandlineutils\src\CommandLineUtils\Conventions\ExecuteMethodConvention.cs:line 25
--- End of stack trace from previous location where exception was thrown ---
   at McMaster.Extensions.CommandLineUtils.CommandLineApplication.<>c__DisplayClass126_0.<OnExecute>b__0() in C:\projects\commandlineutils\src\CommandLineUtils\CommandLineApplication.cs:line 505
   at DotNetOutdated.Program.Main(String[] args) in C:\projects\dotnet-outdated\src\DotNetOutdated\Program.cs:line 73
dotnet outdated 134

**Comment by karl-sjogren** [comment]
We are seeing the same thing on OSX. Works fine if we reinstall with version 1.3 of the tool though.

```
.NET Core SDK (reflecting any global.json):
 Version:   2.1.300
 Commit:    adab45bf0c

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.13
 OS Platform: Darwin
 RID:         osx.10.13-x64
 Base Path:   /usr/local/share/dotnet/sdk/2.1.300/

Host (useful for support):
  Version: 2.1.0
  Commit:  caa7b7e2ba
```

**Comment by gitorrin** [comment]
@karl-sjogren , can you please elaborate on 'reinstalling with version 1.3 of the tool'?

**Comment by simsod** [comment]
@gitorrin means he ran 'dotnet tool uninstall -g dotnet-outdated' and then dotnet tool install -g dotnet-outdated -v 1.3.0'

**Comment by karl-sjogren** [comment]
When I had this error I tried it against my current project and a completely new one (dotnet new mvc).

I've updated the SDK to 2.1.403 and installed the latest version of dotnet outdated and now it seems to work properly though. Tested it on several projects built for NetCoreApp2.0, NetCoreApp2.1 and NetStandard2.0 on OSX Mojave and they all worked fine. Even tested on a clean project (dotnet new mvc) and it worked fine so from my point this is resolved.
